python/nav/ipdevpoll/plugins/bridge.py

```diff
@@ -34,9 +37,20 @@ class Bridge(Plugin):
     @defer.inlineCallbacks
     def handle(self):
         bridge = yield get_multibridgemib(self.agent)
+        bridge_address = yield bridge.get_base_bridge_address()
+        self._save_bridge_address(bridge_address)
         baseports = yield bridge.get_baseport_ifindex_map()
         defer.returnValue(self._set_port_numbers(baseports))
 
+    def _save_bridge_address(self, bridge_address):
```

Comment:
Either all the boxes in my test setup had this field, or I must have missed those. Either way it's a small fix and the pull request is updated